Added RGB and RGBA Support + Prefix logic + CPrintToServer

Nobody-x opened the pull request:
I Added RGB and RGBA support like :

- `{#RRGGBB}`
- `{#RRGGBBAA}`

I also added a prefix logic :

By calling `CSetPrefix("{#FF0000}[my Prefix]")` you can add a prefix before all CPrint* function automatically, with the default color and a space between the prefix and the text. Ex :

This `"CSetPrefix("{#FF0000}[my Prefix]")"` become `"{#FF0000}[my Prefix]{default} "`

So the final message looks like :

`CPrintToChatAll("{red}somestuff")` show `[my Prefix] somestuff`

Finally I added `CPrintToServer` to print phrases which are normaly sent with functions like `CPrintToChat` (so with colors) but without any tags automaticaly.

Updated version to 2.1.0

Commit:
Added prefix and support of {#RRGGBB} and {#RRGGBBAA}

Commit:
Added function CPrintToServer to to remove all color tag before printing to server

Commit:
Check that ReplyToCommand is not handle by the client 0 (server)
else use PrintToServer like colors.inc

removed @noreturn since "void" keyword is used

Bara20 commented:
Add CPrintToServer to https://github.com/Bara20/Multi-Colors/blob/master/addons/sourcemod/scripting/multicolors_example.sp
and maybe a new CPrintToChat with a RGB + RGBA example

Commit:
Fixed CPrintToServer index of "any ..." parameter
Removed prefix from CShowActivity, CShowActivityEx and CShowActivity2
Added CClearPrefix to remove prefix
CSetPrefix("") do nothing

Commit:
Added new RGB and RGBA tag, SetPrefix/ClearPrefix and CPrintToServer Command

Nobody-x commented:
I Fixed some errors, and added new example to the multicolors_example.sp

- `sm_addprefix`
- `sm_clearprefix`

Nobody-x commented:
bump =)
